### PR TITLE
uppercase REPO_TOKEN

### PR DESCRIPTION
**Pull request by casperdcl**
Because consistency with naming conventions is nice.

### CODE REVIEW

File: bin/cml-publish.test.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ describe('CML e2e', () => {
         --repo            Specifies the repo to be used. If not specified is extracted
                           from the CI ENV.
         --token           Personal access token to be used. If not specified in
-                          extracted from ENV repo_token or GITLAB_TOKEN.
+                          extracted from ENV REPO_TOKEN or GITLAB_TOKEN.
```
Comment (casperdcl) [inline bin/cml-publish.test.js:29]:
```suggestion
                          extracted from ENV REPO_TOKEN.
```

File: bin/cml-publish.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ const argv = yargs
   .default('token')
   .describe(
     'token',
-    'Personal access token to be used. If not specified in extracted from ENV repo_token or GITLAB_TOKEN.'
+    'Personal access token to be used. If not specified in extracted from ENV REPO_TOKEN or GITLAB_TOKEN.'
```
Comment (casperdcl) [inline bin/cml-publish.js:74]:
```suggestion
    'Personal access token to be used. If not specified in extracted from ENV REPO_TOKEN, GITLAB_TOKEN, GITHUB_TOKEN, or BITBUCKET_TOKEN.'
```

**Review by casperdcl — COMMENTED** [review]
maybe document alternatives in just one place?

**Comment by 0x2b3bfa0** [comment]
Yes, it would be much better.

**Review by 0x2b3bfa0 — APPROVED** [review]

**Pull request merged**